winterfell/pdf-filler/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.3.3/apache-maven-3.3.3-bin.zip
```

Comment:
I don't think we want .mvn contents in our repo.
We should add it to our git ignore